### PR TITLE
Added image rendering

### PR DESCRIPTION
**Pull request by bitboxer**
This PR adds simple image rendering. It will try to display the image with the correct aspect ratio if only height is given.

### CODE REVIEW

File: src/components/ImageView.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const { element, wrapperStyle, style } = props
+
+  const flattenedStyle = StyleSheet.flatten(style) as ImageStyle
+
+  const width =
+    typeof flattenedStyle?.width === 'number' ? flattenedStyle.width : 300
+  const aspectRatio = element.dimensions.width / element.dimensions.height
+  const height = width / aspectRatio
+
+  console.log('wrapper', wrapperStyle)
```
Comment (Spoutnik97) [inline src/components/ImageView.tsx]: Remove the console log

File: src/services.tsx [hunk 2]
```diff
@@ -136,11 +140,21 @@ function serializeSpan(
   return null
 }
 
-function serializeImage(element: SpanType, key: string) {
+function serializeImage(
+  element: ImageType,
+  key: string,
+  wrapperStyle: StyleProp<ViewStyle>,
```
Comment (Spoutnik97) [inline src/services.tsx]: It should be an optional argument

File: src/services.tsx [hunk 3]
```diff
@@ -136,11 +140,21 @@ function serializeSpan(
...
+  style: StyleProp<ImageStyle>
```
Comment (Spoutnik97) [inline src/services.tsx]: It should be optional too

**Review by Spoutnik97 — CHANGES_REQUESTED** [review]

**Comment by bitboxer** [comment]
Updated the PR 🥳

**Review by Spoutnik97 — APPROVED** [review]

**Pull request merged**